cuckoo/cuckoo_main.py

```diff
@@ -1775,6 +1774,25 @@ def _cleanup_leftovers() -> None:
             if any(leftover_file_name in file_path for leftover_file_name in ["_console_output", "_encrypted_buffer_"]):
                 os.remove(file_path)
 
+    def _get_machine_by_name(self, machine_name) -> Optional[Dict[str, Any]]:
+        """
+        This method grabs the machine info (if it exists) of a machine that matches the requested name
+        :param machine_name: The name of the machine that we want the information for
+        :return: The information about the requested machine
+        """
+        machine_name_exists = False
+        machine: Optional[Dict[str, Any]] = None
+        machines = [machine for host in self.hosts for machine in host["machines"]]
+        for machine in machines:
```

Comment:
simpler statement?:
`machine_name_exists = any(machine_name == machine['name'] for machine in machines)`

Reply:
Unfortunately, since I need the `machine` variable later in this method, I can't iterate over `machines` in a loop that is scoped to the `any` function.